packages/react-scripts/config/paths.js

```diff
@@ -78,9 +78,12 @@ module.exports = {
   dotenv: resolveApp('.env'),
   appPath: resolveApp('.'),
   appBuild: resolveApp('build'),
+  devAppBuild: resolveApp('dev'),
   appPublic: resolveApp('public'),
-  appHtml: resolveApp('public/index.html'),
+  appPopupHtml: resolveApp('public/popup.html'),
   appIndexJs: resolveModule(resolveApp, 'src/index'),
+  appBackgroundJs: resolveModule(resolveApp, 'src/background/index'),
+  appContentScriptJs: resolveModule(resolveApp, 'src/contentScripts/index'),
```

Comment:
This is to support this packages convention of the supported entry points:
- [background script](https://developer.chrome.com/extensions/background_pages) 
- [content script](https://developer.chrome.com/extensions/content_scripts)